test/test_hpytype.py

```diff
@@ -1102,10 +1102,12 @@ def test_call_invalid_specs(self):
             @EXPORT(create_vcall)
             @INIT
         """)
-        with pytest.raises(TypeError):
+        with pytest.raises(TypeError) as err:
             mod.create_var_type()
-        with pytest.raises(TypeError):
+        assert "Cannot use HPy call protocol with var" in str(err.value)
+        with pytest.raises(TypeError) as err:
             mod.create_call_and_vectorcalloffset_type()
+        # assert "Cannot have HPy_tp_call and explicit member" in str(err.value)
```

Comment:
left-over?

Reply:
There is a difference in the parsing code between PyPy and CPython, so the error is raised but differently. I am leaving this here so someone who cares deeply can try to fix it all to be consistent. I don't remember off the top of my head what the different messages are.